### PR TITLE
May 2023 update

### PR DESCRIPTION
**Pull request by shuvashish76**
~Yalp Store~ - inactive
~LTE Cleaner~ - https://github.com/TheRedSpy15/LTECleanerFOSS/discussions/321
~Bromite~ - inactive (check their issue tracker) | also https://github.com/uazo/bromite-buildtools/issues/182
Added new categories - URL Manipulation, Email Aliases
Added new apps - Accrescent, Jerboa, Simple Contacts Pro SE, AnonAddy Android, SimpleLogin, Thumb-Key, Cheogram, Moshidon, jtx Board, Léon, LinkSheet, Tarnhelm, Wall You, Zade's Wallpaper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,13 +206,14 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Connect You**](https://github.com/Bnyro/ConnectYou)
 * [**OpenContacts**](https://gitlab.com/sultanahamer/OpenContacts) <sup>**[[F-Droid](https://f-droid.org/app/opencontacts.open.com.opencontacts)]**</sup>
 * [**Simple Contacts**](https://github.com/SimpleMobileTools/Simple-Contacts) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.contacts.pro)]**</sup>
+    * [**Simple Contacts Pro SE**](https://github.com/stephanritscher/Simple-Contacts) <sup>**[[F-Droid](https://www.f-droid.org/app/de.ritscher.simplemobiletools.contacts.pro)]**</sup>
```
Comment (offa) [inline README.md:209]: I wonder why the fork dev doesn't upstream the changes, but that's another story :man_shrugging:.

**Review by offa — APPROVED** [review]
Thanks for this huge update! :+1:

File: README.md (hunk 1 above)
Comment (shuvashish76) [reply README.md:209]: Main discussion thread : https://github.com/SimpleMobileTools/Simple-Contacts/issues/364

PR rejected : https://github.com/SimpleMobileTools/Simple-Contacts/pull/156

Comment (offa) [reply README.md:209]: Thanks!

**Comment by shuvashish76** [comment]
> Thanks for this huge update! +1

BTW, I added them one at a time whenever I noticed/discovered something but now I realize multiple commits to a single file looks bit spammy. Next time I'll note down them & try to add them all in a single commit.

**Pull request merged**